observability/pillars/dashboard.md

```diff
@@ -0,0 +1,58 @@
+5. Observe the dashboard that is built. Is the data reflecting what the stakeholders set out to answer? If not, make adjustments along they way.
+
+Always remember this process takes time. Building dashboard is easy, building an observable dashboard to show pattern is hard.
+
+## Recommended Tools
+
+- [Azure Monitor Workbooks](https://docs.microsoft.com/en-us/azure/azure-monitor/platform/workbooks-overview) - With Azure workbook, creating a unified interactive experience is easier. This has tight integration with Azure services making this higly customizable without extra tool.
+- [Create dashboard using log query](https://docs.microsoft.com/en-us/azure/azure-monitor/learn/tutorial-logs-dashboards) - Dashboard can be created using log query on Log Analytics data.
+- [Building dashboards using Application Insights](https://docs.microsoft.com/en-us/azure/azure-monitor/learn/tutorial-app-dashboards) - Dashboards can be created using Application Insights as well.
+- [Power Bi](https://docs.microsoft.com/en-us/power-bi/create-reports/service-dashboard-create) - Power Bi is one of the easier tools to create dashboards from data sources and reports.
+- [Grafana](https://grafana.com/tutorials/) - Getting started with Grafana. Grafana is a popular open source tool for dashboarding and visualization.
+- [Azure Monitor as Grafana data source](https://grafana.com/grafana/plugins/grafana-azure-monitor-datasource) - This provides a step by step integration of Azure Monitor to Grafana. 
+
+There are many options available for visualization and dashboarding. Depending on the needs, requirements and implementation, visualization tool used might vary but the overall concept on observability would be more or less the same.
+
+[Brief comparison of various tools](https://docs.microsoft.com/en-us/azure/azure-monitor/visualizations) 
+
+Notes:
+1. Some useful observability features available on Azure Workbooks to leverage on:
+- Performance analysis - A measurement on how the system performs. Workbook template available in gallery.
+- Failure analysis - A report about system failure with details. Workbook template available in gallery.
+- Application Performance Index([Appdex](https://en.wikipedia.org/wiki/Apdex)) - This is a way to measure user satisfaction. It classifies performance into three zones based on a baseline performance threshold T. The template for Appdex is available in Azure Workbooks gallery as well. 
+
+2. Application Insights has various ready to go analysis on observability on a system as well.
+- [User retention analysis](https://docs.microsoft.com/en-us/azure/azure-monitor/app/usage-retention)
+- [User navigation patterns analysis](https://docs.microsoft.com/en-us/azure/azure-monitor/app/usage-flows)
+- [User session analysis](https://docs.microsoft.com/en-us/azure/azure-monitor/learn/tutorial-users)
```

Comment:
I feel like this "Notes" section is too specific to Azure technologies, which is not one of the main goals with the playbook. I would trim some of that or remove this section. 

The one thing I found valuable was the Appdex mention, but I'm not sure if this is the right context for it.

Reply:
I would keep the Azure stuff mainly other options and comparisons have been provided. Assuming a new dev joins CSE and refers to this, he/she would be able to quickly start with microsoft technologies. If there's already a tech preference, this doc should be referred as a guide to concepts mentioned in other sections. Also the various tools provided should be enough to start.